app/assets/javascripts/app/reservation_time_checker.coffee

```diff
@@ -1,8 +1,12 @@
 class window.ReservationTimeChecker
   constructor: (@selector, @alertId = 'duration-alert')->
-    @initAlert()
-    @respondToChange()
+    if @validPage()
+      @initAlert()
+      @respondToChange()
 
+  validPage: ->
+    # These variable are not set on the admin reservation pages
+    reserveInterval? && reserveMinimum? && reserveMaximum?
```

Comment:
I actually don't like being that specific because I'm not really testing whether I'm on the customer reservation page, I'm actually testing if the page I'm on has set up the proper variables and environment in order for this code to function.